fpsim/defaults.py

```diff
@@ -288,45 +288,41 @@ def register_location(name, location_ref):
     'wq4',
     'wq5',
     'nonpostpartum',
-    'total_women_fecund',
-    'method_failures',
-    'birthday_fraction',
-    'short_intervals',
-    'secondary_births',
     'proportion_short_interval',
+
     # Education
     'edu_objective',
     'edu_attainment',
-    # Empowerment and intent: all zero unless using an empowerment module
-    'perc_contra_intent',
-    'perc_fertil_intent',
-    'paid_employment',
-    'decision_wages',
-    'decide_spending_partner',
-    "buy_decision_major", 
-    "buy_decision_daily", 
-    "buy_decision_clothes", 
-    "decision_health",
-    "has_savings", 
-    "has_fin_knowl", 
-    "has_fin_goals",
-    "financial_autonomy", 
-    "decision_making",
-    )
 
+    # Empowerment and intent: all zero unless using an empowerment module
+    # Todo move these to the empowerment module results section
+    # 'perc_contra_intent',
+    # 'perc_fertil_intent',
+    # 'paid_employment',
+    # 'decision_wages',
+    # 'decide_spending_partner',
+    # "buy_decision_major",
+    # "buy_decision_daily",
+    # "buy_decision_clothes",
+    # "decision_health",
+    # "has_savings",
+    # "has_fin_knowl",
+    # "has_fin_goals",
+    # "financial_autonomy",
+    # "decision_making",
+)
 
-for age_group in age_bin_map.keys():
-    array_results += 'total_births_' + age_group
-    array_results += 'total_women_' + age_group
 
-list_results = sc.autolist(
+# list results are results that aren't recorded each time step or have variable lengths.
+# These will NOT be scaled by default!
+float_annual_results = sc.autolist(
+    'pop_size',
     'tfr_years',
     'tfr_rates',
-    'pop_size',
     'mcpr_by_year',
     'cpr_by_year',
-    'contra_access_over_year',
-    'new_users_over_year',
+    # 'contra_access_over_year', # these all seem to be empowerment related? By year values can be generated by resampling the related per-timestep results
```

Comment:
Yeah I think all the `by_year` results will ultimately be able to be dropped, as we can resample using built-in Starsim methods